### PR TITLE
Added support for 6.0.4

### PR DESCRIPTION
**Pull request by alexbacchin**
Just added support to 6.0.4

### CODE REVIEW

File: libraries/helpers.rb [hunk 1]
```diff
@@ -275,6 +275,10 @@ def confluence_checksum_map
           'x64' => 'febd2c6395f34c5ebf7538b1134500aac2dfe87c693f079bcd0802c64092693f',
           'tar' => '67dd59a113472b3a1c9a4385711be0e1f959046db1f64ce19eba8b8dc5d2e535',
         },
+        '6.0.4' => {
+            'x64' => '7bc9001af3e75712f1ea0bcf4f3a9261186f95d0d50a5bc1e8d236acf2c5a88d',
```
Comment (legal90) [inline libraries/helpers.rb]: Could you please fix the indentation (to 2 spaces), to fix TravisCI build?

**Review by legal90 — CHANGES_REQUESTED** [review]
Thanks! But we need a green light from TravisCI.

**Comment by alexbacchin** [comment]
Done

**Review by legal90 — APPROVED** [review]

**Comment by legal90** [comment]
Thanks!

**Pull request merged**